python/llm/src/bigdl/llm/cmd/cmd-cli.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+# Default values
+threads=28
+n_predict=-1
+interactive=false
+model_family=""
+prompt=empty
+ctx_size=512
+
+script_dir="$(dirname "$(readlink -f "$0")")"
+lib_dir="$(dirname "$script_dir")/libs"
+
+# Function to display help message
+function display_help {
+  echo "Usage: ./script.sh [OPTIONS]"
+  echo "Options:"
+  echo "  -c, --ctx_size N      size of the prompt context (default: 512)"
+  echo "  -i, --interactive     run in interactive mode"
+  echo "  -n, --n_predict N     number of tokens to predict (default: -1, -1 = infinity)"
+  echo "  -m, --model FNAME     model path"
+  echo "  -p, --prompt PROMPT   prompt to start generation with (default: empty)"
+  echo "  -t, --threads N       number of threads to use during computation (default: 28)"
+  echo "  -x, --model_family {llama,bloomz,gptneox}"
+  echo "                        family name of model"
+  echo "  -h, --help            display this help message"
+  exit 1
+}
```

Comment:
It's not clear to me why we want to limit what parameters of `llama.cpp` that we want to support here; in real world, people are using many different parameters to tune the model behavior (such as temperature, repetition penalty, top_p, top_k, etc.; see an example [here](https://www.reddit.com/r/LocalLLaMA/comments/1343bgz/what_model_parameters_is_everyone_using/)). I think we should simple expose all the possible parameters.